install/kubernetes/helm/istio/charts/tracing/values.yaml

```diff
@@ -40,3 +40,4 @@ ingress:
     # - secretName: tracing-tls
     #   hosts:
     #     - tracing.local
+gatewayName: ingressgateway
```

Comment:
seems like overkill, can be cleaned up across the code base in one PR.

Reply:
@sdake +1 to kill this, will propose a PR soon.